### ISSUE TITLE
0.9.0 pip package installation failure

### ISSUE THREAD

**Issue by unrepentantgeek** [enthought/sat-solver#289]
The 0.9.0 pip package won't install on Debian stable and testing:
```
$ python3 -m venv venv
$ source venv/bin/activate
$ pip install simplesat
Collecting simplesat
  Downloading simplesat-0.9.0-py3-none-any.whl (248 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 248.8/248.8 kB 6.5 MB/s eta 0:00:00
ERROR: Error decoding metadata for /tmp/pip-unpack-cs6ba74i/simplesat-0.9.0-py3-none-any.whl: 'utf-8' codec can't decode byte 0x82 in position 1851: invalid start byte in METADATA file
```

**Comment by itziakos** [comment]
duplicate of #286 (and fixed in the maitenance/0.9.x branch) will try to get a new release on Monday.

**Comment by itziakos** [comment]
Reopened #286 to avoid confusion
